openreview/arr/webfield/programChairsWebfield.js

```diff
@@ -170,6 +170,53 @@ return {
       `
     },
     reviewerEmailFuncs: [
+      {
+        label: 'Reviewers with No Rebuttal Responses', filterFunc: `
+        if (row.notesInfo.length <= 0){
+          return false;
+        }
+
+        const buildReplyToMap = (note) => {
+          const replyToMap = new Map();
+          (note?.details?.replies ?? []).forEach((reply) => {
+            if (!replyToMap.has(reply.replyto)) {
+              replyToMap.set(reply.replyto, []);
+            }
+            replyToMap.get(reply.replyto).push(reply);
+          });
+          return replyToMap;
+        }
+        
+        const childrenWithSignature = (reply, signature, replyToMap) => {
+          const queue = replyToMap.get(reply.id) ?? [];
+          const matchingReplies = [];
+          const count = 0;
+          while (queue.length > 0) {
+            const nextReply = queue.pop();
+            if (nextReply.signatures[0].includes(signature) && nextReply.readers.some(reader => reader.includes('/Authors')) && nextReply.readers.some(reader => reader.includes('/Reviewers'))) {
+              matchingReplies.push(nextReply);
+            }
+            queue.push(...(replyToMap.get(nextReply.id) ?? []));
+          }
+          return matchingReplies;
+        }
+
+        return row.notesInfo.some(noteObj =>{
+          const noteReplyMap = buildReplyToMap(noteObj.note);
+          const officialReview = noteObj?.officialReview
+          if (!officialReview) {
+            return false;
+          }
+          const anonId = officialReview.anonymousId;
+          const authorReplies = childrenWithSignature(officialReview, '/Authors', noteReplyMap);
+          if (authorReplies.length <= 0) {
+            return false;
+          }
+          const reviewerResponses = childrenWithSignature(officialReview, anonId, noteReplyMap);
```

Comment:
A simpler way of doing this is to find all the replies in the forum where the signature is an author. For each reply check there is a direct reply made by the reviewer. This is tricky because some authors post 3 comments for their rebuttal given the character limit and the reviewer will reply to one of them only. How do you solve that?